src/components/inline/InputText/InputText.stories.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+
+import InputText from "./InputText";
+
+const demoStyling = {
+  left: "50%",
+  position: "absolute",
+  top: "50%",
+  transform: "translate(-50%, -50%)",
+  width: "200px",
+};
+
+export default {
+  title: "Inline/InputText",
+  component: InputText,
+};
+
+const [text, setValue] = useState("");
+
+const Template = ({ placeholderText, value, type }) => {
+  const handleChange = (e) => {
+    setValue(e.target.value);
+  };
+
+  return (
+    <span style={demoStyling}>
+      <InputText
+        type={type}
+        placeholderText={placeholderText}
+        onChange={handleChange}
+        value={value}
+      />
+    </span>
+  );
+};
+
+export const DefaultInputText = Template.bind({});
+DefaultInputText.args = {
+  placeholderText: "email address",
+  value: text,
```

Comment:
This doesn't really make any sense as a Storybook argument. This should be something that's linked to state, and there's no reason for a Storybook user to try to set this value from the Storybook controls panel.

Reply:
ok